debian/copyright

```diff
@@ -0,0 +1,50 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: revpi-tools
+Upstream-Contact: KUNBUS GmbH <support@kunbus.com>
+Source: https://github.com/RevolutionPi/revpi-tools
+
+Files:     *
+Copyright: 2016-2022 Kunbus GmbH
+License:   GPL-2+
+
+Files:     examples-watchdog/*
+Copyright: 2019-2022 KUNBUS GmbH
+License:   MIT
```

Comment:
Should we always put license text to the end of file and use the short name in this blocks?